### PR TITLE
[regression test](schema change) add case for tinyint/smallint/int/bigint/float/double type in agg

### PR DESCRIPTION
**Pull request by liugddx**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by liugddx** [comment]
run buildall

**Comment by liugddx** [comment]
run pipelinex_p0

File: regression-test/suites/schema_change_p0/test_schema_change_agg_check_all_types.groovy [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+    }
+
+    sql """ DROP TABLE IF EXISTS ${tableName3} """
+
+    sql """
+    CREATE TABLE IF NOT EXISTS ${tableName3} (
+      `k1` int(11) NULL,
+      `k2` tinyint(4) NULL,
+      `k3` smallint(6) NULL,
+      `k4` int(30) sum NULL,
```
Comment (Lchangliang) [inline regression-test/suites/schema_change_p0/test_schema_change_agg_check_all_types.groovy:38]: Can `k4` be modified to other type?

Comment (liugddx) [reply regression-test/suites/schema_change_p0/test_schema_change_agg_check_all_types.groovy:38]: yes

**Review by Lchangliang — APPROVED** [review]

**Comment by Lchangliang** [comment]
LGTM

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**